nautobot-app/{{ cookiecutter.project_slug }}/docs/admin/install.md

```diff
@@ -21,7 +21,7 @@ Here you will find detailed instructions on how to **install** and **configure**
 ## Install Guide
 
 !!! note
-    Apps can be installed manually or using Python's `pip`. See the [nautobot documentation](https://nautobot.readthedocs.io/en/latest/plugins/#install-the-package) for more details. The pip package name for this app is [`{{ cookiecutter.app_slug }}`](https://pypi.org/project/{{ cookiecutter.app_slug }}/).
+    Apps can be installed manually or using Python's `pip`. See the [nautobot documentation](https://docs.nautobot.com/projects/core/en/stable/plugins/#install-the-package) for more details. The pip package name for this app is [`{{ cookiecutter.app_slug }}`](https://pypi.org/project/{{ cookiecutter.app_slug }}/).
```

Comment:
The updated link redirects to https://docs.nautobot.com/projects/core/en/stable/development/apps/#install-the-package, but I think a better link would be https://docs.nautobot.com/projects/core/en/stable/user-guide/administration/installation/app-install/ instead.